README.md

```diff
@@ -189,7 +193,88 @@ A breakdown of the options and methods available to the SDK.
 
 - **`onComplete {Function} optional`**
 
-  Callback that fires when both the document and face have successfully been captured. It returns an object that contains both captures. This event data should sent to your backend where the full API requests will be made.
+  Callback that fires when both the document and face have successfully been captured. It returns an object that contains both captures. This event data should sent to your backend where the full [API requests](#uploading-files-to-onfido-api) will be made.
+
+  Here is an `onComplete` callback example, making use of the `getCaptures` convenience function:
```

Comment:
We should fully document the other public methods of the API:

- tearDown
- getCaptures